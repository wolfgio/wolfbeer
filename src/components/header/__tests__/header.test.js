import React from 'react';
import { create, act } from 'react-test-renderer';

import Header from '../header';

describe('<Header />', () => {
  it('should render without error', () => {
    let root;
    act(() => {
      root = create(<Header />);
    });

    expect(root.toJSON()).toMatchSnapshot();
  });
});
