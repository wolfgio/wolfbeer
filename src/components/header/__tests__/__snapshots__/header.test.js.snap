// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render without error 1`] = `
<header
  className="sc-AykKC kQjYic"
>
  <div
    className="sc-AykKD kZMYJW"
  >
    <img
      className="sc-AykKE cvsQeC"
      src={
        Object {
          "process": [Function],
        }
      }
    />
    <div
      className="sc-AykKF evZlmn"
    >
      <h1>
        Wolfbeer
      </h1>
      <small>
        Delivery
      </small>
    </div>
  </div>
</header>
`;
